### PR TITLE
Fix dependencies to allow builds in 2023

### PR DESCRIPTION
**Pull request by ekacnet**
Right out of the box `cargo build` don't work for `clean_xar_mounts` in 2023.
It turns out that something is wrong with the 2.1.x version of slog, I didn't look very hard to know what. It seems that upgrading to something a bit newer is working.
Similarly using 1.0 version of anyhow is not working (complains about missing backtrace) instead using a recent version and enabling the `backtrace` feature is solving the problem

### CODE REVIEW

File: xar/clean_xar_mounts/Cargo.toml [hunk 1]
```diff
@@ -5,12 +5,12 @@ authors = ["chip <chip@fb.com>"]
 edition = "2021"
 
 [dependencies]
-anyhow = "1.0"
+anyhow = { version = "1.0.52", features = ["backtrace"] }
```
Comment (itamaro) [inline xar/clean_xar_mounts/Cargo.toml:8]:
```suggestion
anyhow = { version = "1.0.75", features = ["backtrace"] }
```

File: xar/clean_xar_mounts/Cargo.toml [hunk 2]
```diff
@@ -5,12 +5,12 @@ authors = ["chip <chip@fb.com>"]
...
 clap = "2.31"
 filetime = "0.2.8"
 lazy_static = "1.0.*"
 nix = "0.20.2"
 regex = "1.5.5"
-slog = "2.1.*"
+slog = "2.3.0"
```
Comment (itamaro) [inline xar/clean_xar_mounts/Cargo.toml:14]:
```suggestion
slog = "2.7.*"
```

**Review by itamaro — CHANGES_REQUESTED** [review]
I suggested changes to match the versions we build with internally - please let me know if this works for the open-source build

**Comment by ekacnet** [comment]
Yes @itamaro  it's working I build it locally.

**Pull request closed without merging**